.github/workflows/tags.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    tags: 'v*'
```

Comment:
If we'd like to bump it on every release, we could leverage `release` trigger:

```yaml
on: 
  release:
    types: [published]
```

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#release-event-release